src/m.graph.js

```diff
@@ -669,6 +672,10 @@ please.GraphNode.prototype = {
         // binds, so as long as the calls are sorted, this extra
         // overhead should be insignificant.
         if (this.visible && this.__drawable && typeof(this.draw) === "function") {
+            if (please.gl.ctx === null) {
```

Comment:
the graph's drawing code should never be called at all in 2D mode.  This just introduces an extra branch into something that is called every frame for 3D mode to prevent some behavior in 2D mode.